### PR TITLE
Several new integrations for the ecosystem page

### PR DESCRIPTION
**Pull request by timothyhinrichs**

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -706,6 +706,46 @@ integrations:
...
+    description: Gluu Gateway provides API authentication and authorization for websites built on Kong.  Gluu provides an OPA plugin to handle API authorization.
+    labels:
+      layer: network
+      category: gateway
+    software:
+      - gluu
+      - kong
+    code:
+      - https://github.com/GluuFederation/gluu-gateway
+    docs:
```
Comment (ashutosh-narkar) [inline docs/website/data/integrations.yaml:746]: `docs` -> `tutorials`. There is no check for the `docs` title so this won't show up.

Comment (timothyhinrichs) [reply docs/website/data/integrations.yaml]: Removed

**Comment by timothyhinrichs** [comment]
Went back and forth. Without some indicator as to the rank, the ordering is
totally mysterious.  Usually there are stars or the like but that's a bit
premature.  Any ideas?

**Comment by patrick-east** [comment]
I could go either way on it. If we do keep it I think adding a space between the name and number would maybe look a little better. 

I wonder though how much information someone would really get from seeing the actual numbers, without reading the code its unclear what the numbers are, eg

image

19 what? Admission controllers? Version 19?

That being said I don't really have any better ideas..

@peterstyra got any thoughts?

**Comment by peterstyra** [comment]
Definitely should identify what the number is:
Kubernets Admission Control (<Whatever_the_heck_it_is> 19)
Kubernets Admission Control (Rank 19) or
Kubernets Admission Control (Ranked 19 out of 100)

**Comment by timothyhinrichs** [comment]
What if we just listed out each of the components that make up the ranking
(and leave off the total ranking).
K8s admission control
10 videos
3 blogs
2 codebases

**Comment by peterstyra** [comment]
Now I've discovered the ecosystems page, I have more context - yay!

What's the purpose of adding this information to an integration card? How will that help the user to know they want to click on a card?

**Comment by timothyhinrichs** [comment]
Dropped the annotation on each card.  Instead just added a note saying how the cards are sorted.

**Comment by timothyhinrichs** [comment]
Added some missing images as well.  

If anyone wants me to split off the content-sorting into its own PR let me know.  The remaining changes are just so simple it didn't seem worth it.

**Comment by patrick-east** [comment]
Looks pretty good, one nit pick with the image for SQL filtering:

image
Looks like the transparent background got lost at some point, do we have another one we can use?

**Comment by timothyhinrichs** [comment]
Uploaded new image--one not derived from a JPG

**Comment by patrick-east** [comment]
Still doesn't look quite right, not sure what the source is/licensing for it.. but a few minutes in gimp fiddling with the alpha channel gives:

sql-datafiltering

If its OK for us to modify/use the modified version we can just use that one.

**Comment by timothyhinrichs** [comment]
Thanks for fixing that image up @patrick-east.  Included it in this PR.

**Pull request merged**